tests/goss-linux.yaml

```diff
@@ -17,6 +17,9 @@ command:
   datadog-agent:
     exec: datadog-agent version
     exit-status: 0
+  default_java:
+    exec: java --version
+    exit-status: 0
```

Comment:
This test is failing (Linux only) with the following error:

```
[stderr]  sh: 1: java: not found
```

=> preventing the `goss-common` test to be run (fail fast)